requirements/e2e_requirements.txt

```diff
@@ -7,3 +7,4 @@ pytest-tap
 pytorch-lightning
 torchvision
 sklearn
+git+https://github.com/neptune-ai/transformers.git@rj/proper-run-lifecycle#egg=transformers
```

Comment:
Could you add comment that we should replace it once we finish developing HF integration?